.eslintrc.js

```diff
@@ -1,6 +1,39 @@
 module.exports = {
-  globals: {
-    __PATH_PREFIX__: true,
+  env: {
+    browser: true,
+    jest: true,
   },
-  extends: `react-app`,
-}
\ No newline at end of file
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'semistandard',
+    'prettier',
+    'plugin:prettier/recommended',
+    'plugin:import/typescript',
```

Comment:
I was actually talking about https://github.com/lydell/eslint-plugin-simple-import-sort#usage, not eslint-plugin-import (which afaict does not sort imports).